app/controllers/api/v2/template_controller.rb

```diff
@@ -22,6 +23,28 @@ def import
         }).import!
         render :json => { :message => results }
       end
+
+      api :POST, "/template/export", N_("Initiate Export")
+      param :verbose, :bool, :required => false, :desc => N_("Set verbosity of export")
+      param :repo, String, :required => false, :desc => N_("Override the default repo from settings")
+      param :branch, String, :required => false, :desc => N_("Branch in Git repo.")
+      param :filter, String, :required => false, :desc => N_("Export templates with names matching this regex (case-insensitive; snippets are not filtered).")
+      param :negate, :bool, :required => false, :desc => N_("Negate the prefix (for purging).")
+      param :export_mode, String, :required => false, :desc => N_("Specify how to handle metadata ('refresh', 'remove', 'keep')")
```

Comment:
could you also use enum type here and where applicable (ideally in import api), just pass an array of values, one example can be found at https://github.com/theforeman/foreman_virt_who_configure/blob/master/app/controllers/foreman_virt_who_configure/api/v2/configs_controller.rb#L40

this will result in better /apidoc